### PR TITLE
[npo] added npo3 support

### PR DESCRIPTION
**Pull request by dubber0**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [ ] Improvement
- [ ] New extractor
- [x] New feature

---

### Description of your *pull request* and other information
Some NPO videos are also available at npo3.nl, I've added support for that URL and wrote a test for it. The videos at npo3.nl can be downloaded exactly the same way as on npo.nl, so I only had to add a regex. npo3.nl is an official site of NPO.

### CODE REVIEW

File: youtube_dl/extractor/npo.py [hunk 1]
```diff
@@ -28,14 +28,15 @@ def _get_token(self, video_id):
...
-    IE_DESC = 'npo.nl and ntr.nl'
+    IE_DESC = 'npo.nl, npo3.nl and ntr.nl'
     _VALID_URL = r'''(?x)
                     (?:
                         npo:|
                         https?://
                             (?:www\.)?
                             (?:
                                 npo\.nl/(?!(?:live|radio)/)(?:[^/]+/){2}|
+                                npo3\.nl/(?!/)(?:[^/]+/){2}|
```
Comment (dstftw) [inline youtube_dl/extractor/npo.py]: `(?!/)` makes no sense.

File: youtube_dl/extractor/npo.py [hunk 2]
```diff
@@ -28,14 +28,15 @@ def _get_token(self, video_id):
...
                                 ntr\.nl/(?:[^/]+/){2,}|
```
Comment (dstftw) [inline youtube_dl/extractor/npo.py]: New domain should be embedded here.

File: youtube_dl/extractor/npo.py [hunk 3]
```diff
@@ -89,6 +90,17 @@ class NPOIE(NPOBaseIE):
...
+        'url': 'https://www.npo3.nl/3onderzoekt/16-09-2015/VPWON_1239870',
+        'md5': '5196e6b895a779445a610f1f4c4145e7',
+        'info_dict': {
+            'id': 'VPWON_1239870',
+            'ext': 'm4v',
+            'title': '3Onderzoekt: Krakers',
+            'description': 'md5:0f79d45e282659ba4be99b125196b6af',
+            'upload_date': '20150916',
+            'duration': 1514,
+        },
```
Comment (dstftw) [inline youtube_dl/extractor/npo.py]: Make `only_matching`.

**Comment by dubber0** [comment]
What do you mean with embedded?

**Pull request merged**